client/src/main/java/org/evosuite/coverage/mutation/Mutation.java

```diff
@@ -47,11 +50,11 @@ public class Mutation implements Comparable<Mutation> {
 
     private final String mutationName;
 
-    private final BytecodeInstruction original;
+    private final transient BytecodeInstruction original;
 
-    private final InsnList mutation;
+    private final transient InsnList mutation;
 
-    private final InsnList infection;
+    private final transient InsnList infection;
```

Comment:
Why do all these fields need to be `transient`?

Reply:
We had to make the "Mutation" class Serializable. In turn, we made these 3 fields transient because the respective classes were not Serializable (and we decided that it didn't make sense to make them Serializable).